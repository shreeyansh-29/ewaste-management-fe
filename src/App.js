import React from "react";
import "bootstrap/dist/css/bootstrap.min.css";
import AllRoutes from "./routes/routes";
import store from "./redux/store";
import {Provider} from "react-redux";

function App() {
  return (
    <Provider store={store}>
      <AllRoutes />
    </Provider>
  );
}
export default App;
