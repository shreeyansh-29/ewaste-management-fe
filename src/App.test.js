/* eslint-disable no-undef */
import React from "react";
import App from "./App";
import Enzyme, {shallow} from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import toJson from "enzyme-to-json";
import configureStore from "redux-mock-store";

Enzyme.configure({adapter: new Adapter()});
const mockStore = configureStore([]);

describe("App", () => {
  let wrapper;
  let store;
  store = mockStore({
    forgotPassword: {},
    resetPasssword: {},
    signIn: {},
    signUp: {},
  });
  beforeEach(() => {
    wrapper = shallow(<App store={store} />);
  });
  test("test App", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
