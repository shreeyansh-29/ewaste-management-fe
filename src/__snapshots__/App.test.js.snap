// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test App 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "asyncReducers": Object {},
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Routes />
</Provider>
`;
